extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -208,7 +208,7 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   isLoopingEligible() {
-    return this.noOfSlides_ > 2;
+    return this.noOfSlides_ > 1;
```

Comment:
Talked offline, lets verify that the `hasPrev` and `hasNext` functions here still behave correctly? =)

Reply:
Made a change to make sure that the slides appear in the correct order. Seems hacky to me. Let me know what you think?

Reply:
Made the condition easier to follow.